src/Mvc/Mvc.Api.Analyzers/src/ActualApiResponseMetadataFactory.cs

```diff
@@ -102,13 +102,13 @@ public static class ActualApiResponseMetadataFactory
                 case ObjectCreationExpressionSyntax creation:
                     {
                         // Read values from 'return new StatusCodeResult(200) case.
-                        var result = InspectMethodArguments(semanticModel, creation, creation.ArgumentList, cancellationToken);
+                        var result = InspectMethodArguments(semanticModel, creation, creation.ArgumentList!, cancellationToken);
                         statusCode = result.statusCode ?? statusCode;
                         returnType = result.returnType;
 
                         // Read values from property assignments e.g. 'return new ObjectResult(...) { StatusCode = 200 }'.
                         // Property assignments override constructor assigned values and defaults.
-                        result = InspectInitializers(symbolCache, semanticModel, creation.Initializer, cancellationToken);
+                        result = InspectInitializers(symbolCache, semanticModel, creation.Initializer!, cancellationToken);
```

Comment:
Same question for Initializer: http://sourceroslyn.io/#Microsoft.CodeAnalysis.CSharp/Generated/Syntax.xml.Syntax.Generated.cs,2850